crates/holochain/src/local_network_tests.rs

```diff
@@ -46,11 +46,19 @@ async fn conductors_call_remote(num_conductors: usize) {
         .into_iter()
         .map(|c| c.into_cells().into_iter().next().unwrap())
         .collect();
+
     conductors.exchange_peer_info().await;
 
+    // Make sure that genesis records are integrated now that conductors have discovered each other. This makes it
+    // more likely that Kitsune knows about all the agents in the network to be able to make remote calls to them.
+    consistency_60s(cells.iter()).await;
```

Comment:
Tests tend to behave much better when conductors are given a chance to start talking before remote calls and signals are made. I also think it's easier to diagnose the failure if something goes wrong here than it is to understand why the test fails if it just goes straight into doing work